composer.json

```diff
@@ -14,12 +14,23 @@
     ],
     "require": {
         "php": "^7.1",
+        "doctrine/dbal": "^2.9",
+        "jdorn/sql-formatter": "^1.2.16",
+        "psr/container": "^1.0",
+        "symfony/config": "^3.4.30|^4.3.3|^5.0",
+        "symfony/dependency-injection": "^3.4.30|^4.3.3|^5.0",
+        "symfony/doctrine-bridge": "^3.4.30|^4.3.3|^5.0",
+        "symfony/framework-bundle": "^3.4.30|^4.3.3|^5.0",
```

Comment:
@stof @alcaeus which Symfony versions do we want to support exactly?

It seems tricky to fully support all 3 major versions. 

See for example this fail: https://travis-ci.org/doctrine/dbal-bundle/jobs/656726538?utm_medium=notification&utm_source=github_status

For now as @stof and me discussed on the symfony hack day I started to move the dbal code from DoctrineBundle `2.x` here. 

So should I use the same symfony requirements as in https://github.com/doctrine/DoctrineBundle/blob/2.0.x/composer.json#L27 ?

Reply:
Since this is building on top of DoctrineBundle 2.0.x, you should use the same requirements as that one.